BTE-Trusted-Entity/sporran-extension
# SK-111 fix: increase the index of the word shown on error by one

## fixes KILTProtocol/ticket#SK-111
Increase the index passed to the error message shown in the `ImportBackupPhrase` by 1 since the array index starts at 0

## How to test:

- Go to the Import Backup Phase page
- Use a false password
- See that the index of the word with the error is correct

## Checklist:
